code/modules/clothing/head/jobs.dm

```diff
@@ -15,6 +15,12 @@
 	dynamic_hair_suffix = ""
 	pocket_storage_component_path = /datum/component/storage/concrete/pockets/chefhat
 	dog_fashion = /datum/dog_fashion/head/chef
+	///the chance that the movements of a mouse inside of this hat get relayed to the human wearing the hat
+	var/mousecontrolprobability = 20
+
+/obj/item/clothing/head/chefhat/iamassumingdirectcontrol
```

Comment:
snake case

Reply:
do we use snake_case for paths? "chefhat" isn't in snake_case...

Reply:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#type-paths-must-be-snake-case

yes, and it's a new thing